plugins/generator-1.20.1/forge-1.20.1/mappings/blockstateproperties.yaml

```diff
@@ -0,0 +1,305 @@
+ATTACHED:
+  - BlockStateProperties.ATTACHED
+  - attached

```

Comment:
Where is mapping table 1 used?

Reply:
Registry names of these properties, I added them that soon for the sake of not rewriting both mappings and templates when custom states land.

Reply:
I see. But where will we need them?